src/shred_network/repair_service.zig

```diff
@@ -110,14 +123,17 @@ pub const RepairService = struct {
         self.report.deinit();
     }
 
-    const min_loop_duration_ns = 100 * std.time.ns_per_ms;
+    const min_loop_duration_ns = 500 * std.time.ns_per_ms;
 
     pub fn run(self: *Self) !void {
         var waiting_for_peers = false;
         var timer = try std.time.Timer.start();
         var last_iteration: u64 = 0;
         while (!self.exit.load(.acquire)) {
-            if (self.sendNecessaryRepairs()) |_| {
+            _ = timer.read();
```

Comment:
this does nothing?

Reply:
403d5f7a71d7597123eae9999bae8913912bfc40